### PR TITLE
Finished +/- button functionality

### PR DESCRIPTION
**Pull request by klutzx92**

### CODE REVIEW

**Comment by timotiusvincent** [comment]
LGTM

**Comment by abishekc** [comment]
**What is the goal of this PR?**
The goal of this PR was to finish the logic behind two buttons, one to “add access” and the other to “remove access” to the assessment that is currently being created.

**Were you able to view screenshots, story cards, and or test the app?**
Yes, we were able to see the application by finding the link on the info.yml sheet. The app does load and does seem to be functioning.

**Is the PR in a shape that makes it easy to review?**
Yes, it seems to be in a bite-sized format, which makes it easy to review and understand the given goal. Function names and #div identifiers were easy to decipher. We quickly found them on the live app using these identifiers. The code was also well commented. The description in the pr was a bit lackluster, but the story on pivotal was much more comprehensive, (though it wasn’t linked, so we had to dig around a bit to find it)

**What is the status of the tests?**
There are no included tests in this pull request. Although the pull request was atomized (i.e. the feature request was small, which is great), there were no included tests to ensure that the feature was indeed working as expected.

**What is the style of the code like?**
Looking at the build configuration file for the Travis run, there doesn’t seem to be any automatic linters that were run for this particular PR. That said, their code does seem to follow a standard format. The status is passing for Travis. As stated above, their code was commented and well organized (except for the placement of the javascript) and was easily readable.

**Is there anything missing from this PR that should be included?**
As mentioned above, there was no testing included - we were only able to see that the feature worked when we went to the heroku deployment and played around with the functionality.

**Feature Design: Are there any parts of this feature that are cause for concern?**
There might be nothing. Could this feature be abused by users? Does the code look safe in how it treats data? Are there any potential bugs that should be guarded against?
There are no bugs we could find, It's all front end, so no data is touched and there is not a lot of room for abuse. 
In terms of design, we were a bit bothered by how the buttons are slightly different sizes. Also you can only remove the most bottommost access, so removing one in the middle would be a pain point as it would require the user to manually switch the form info in it with the bottom most one before deleting. (If the user changed their mind after filling out several access forms).  

---
### Overall Assessment
**What does this PR do well?**
This PR does a few things very well - it focuses on one single feature, which is great, in case something needs to be rolled back or checked. The amount of code committed is less than 30 lines, which made it easy to read and understand. Furthermore, the code was well written and linted properly.

**What is something that could be improved with this PR?**
A couple of things stood out to us when we went over the PR. First, the javascript code was added to the end of the page (and it seems like it’s still there) instead of being apportioned to its own file in the proper place. Furthermore, the functionality did not include any rspec, javascript or acceptance tests in order to ensure that the feature works as intended.

File: app/views/questions/index.html.erb [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <head>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.0.js"></script>
```
Comment (rchunter) [inline app/views/questions/index.html.erb:7]: You might want to use an SRI like
<script
  src="https://code.jquery.com/jquery-3.5.0.min.js"
  integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
  crossorigin="anonymous"></script>

Comment (chrononyan) [reply app/views/questions/index.html.erb:7]: In terms of best practices, it's generally preferred to have scripts loaded `async`/`defer` and/or near the bottom of `<body>` so the page can visually render first while the scripts load in

**Pull request merged**